ipfshttpclient/client/pin.py

```diff
@@ -55,6 +55,8 @@ def ls(self, type="all", **kwargs):
 
```

Comment:
Please extend this `code-block` with an example of `.pin.ls()` with parameters that succeeds and one that doesn't.

Reply:
Added.